src/App.css

```diff
@@ -7,32 +7,30 @@
   pointer-events: none;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
+.images {
+  display: block;
+  max-width:230px;
+  max-height:95px;
+  width: auto;
+  height: auto;
+  margin-left: auto;
+  margin-right: auto;
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
 }
 
-.App-link {
-  color: #61dafb;
-}
+.reviewOnHomepage{
+  background-color: rgb(252, 242, 203);
+  border-style: solid;
+  border-color: rgb(211, 193, 37);
+  text-align: center;
+  
```

Comment:
Consider fixing the width of the review cards - unless the mosaic effect is intentional!
If all the cards are the same width as the longest one you expect to appear they look more uniform